onesixtyone.c

```diff
@@ -61,11 +62,10 @@ char* snmp_errors[] = {
 struct {
 	int debug;
 	int log;
-  int port;
-  int print_ip;
+    int port;
+    int print_ip;
```

Comment:
I honestly prefer spaces. Looks like you could either replace all tabs with 2 spaces, or go with tabs (most common in that file) and fix up the few things that don't match.